config/stub_templates/clojure/stub_config.toml

```
@@ -0,0 +1,26 @@
+name = "clojure"
+source_file_ext = "clj"
+
+preprocessor = "lisp-like"
+
+[type_tokens]
+Int = "Integer/parseInt"
+Long = "Long/parseLong"
+Float = "Float/parseFloat"
+Bool = "not= \"0\""
+String = "NONE"
+Word = "NONE"
+
+[variable_name_options]
+casing = "kebab-case"
```

It could be a nice touch to rename the options to match their meaning ("camelCase", "kebab-case", "snake_case", etc.) in a separate PR but for now it is called "kebab_case".

```suggestion
casing = "kebab_case"
```

---

Good catch, note to self don't split PRs